access/pagerduty/config.go

```diff
@@ -69,6 +74,13 @@ user_email = "me@example.com" # PagerDuty bot user email (Could be admin email)
 [log]
 output = "stderr" # Logger output. Could be "stdout", "stderr" or "/var/lib/teleport/pagerduty.log"
 severity = "INFO" # Logger severity. Could be "INFO", "ERROR", "DEBUG" or "WARN".
+
+# Plugin will only process requests containing specified roles. If contains "*" (default) all requests will be processed.
+[roles]
+allowlist = [
+  "role1", 
+  "role2"
+]
```

Comment:
This section is written into the configuration file when configuring the plugin
It should contain the most reasonable/friendly configuration we can think of
In this case, can you please set the default value to be `[ "*" ]`
Otherwise, users must edit this section